src/MergeContext.js

```diff
@@ -755,8 +951,8 @@ class PullRequest {
         if (this._draftPr())
             throw this._exObviousFailure("just a draft");
 
-        if (!this._messageValid)
-            throw this._exLabeledFailure("invalid commit message", Config.failedDescriptionLabel());
+        if (!this._commitMessage.valid())
+            throw this._exLabeledFailure("invalid commit message: " + this._commitMessage.errorMessage(), Config.failedDescriptionLabel());
```

Comment:
If possible, let's undo this and related changes that create a notion of an "invalid this._commitMessage": When Anubis cannot create a commit message, log the problem and set this._commitMessage to null. The _commitMessage object should either be in a "valid" state or not be stored at all.